megamek/src/megamek/server/Server.java

```diff
@@ -34422,6 +34418,16 @@ else if (bldg.getPhaseCF(coords) != bldg.getCurrentCF(coords)) {
                     }
                 }
             }
+            // if boom is still zero then we found no (explosive) ammo
+            // alter the reported message to suit that situation
+            if (boom === 0) {
```

Comment:
The failure is the usage of `===` instead of `==` for equality.

JavaScript has the triple equals, "very very equal" operator, while Java only has double equals. The running joke being Java and JavaScript are like Car and Carpet.

Reply:
Or ham and hamster. PHP has `===` too. But I guess Java can dispense with such nonsense since it's got a decent type system.

I'm installing Gradle locally so I can get an actual build process up and going. And this is an easy enough fix. Thanks for chiming in.